packages/node-utils/tsconfig.json

```diff
@@ -1,9 +1,6 @@
 {
     "extends": "../../tsconfig.base.json",
-    "compilerOptions": {
-        "skipLibCheck": false,
```

Comment:
we sure we don't want it?

Reply:
I don't think it's useful to check 3rd party files for type errors (it will still use type definitions from these files only ignore errors). https://www.testim.io/blog/typescript-skiplibcheck/

It's off in all other packages anyway.

Reply:
metamask ppl are doing that, they run skipLibCheck false. see https://github.com/trezor/trezor-suite/issues/10389

I am not saying we need that here, just that I encountered issues with this option

Reply:
Yes, I know but for us it's nearly impossible to run `skipLibCheck: false` because of monorepo size a huge amount of clashing types version etc. Only reason why it was enabled in these libraries because they had minimal amount of dependencies so it was possible, because it doesn't conflicted with anything.

There is no real benefit having this enabled in only like 3 minors packages when it's off in all others (and we can't turn it on even if we want). And I am still having doubts that it's useful even if we can turn it on.

Reply:
It can prevent some silent `any` type coming from 3rd party lib but I never encountered it anyway.

Reply:
ok then :P